gobblin-core/src/main/java/gobblin/writer/http/AsyncHttpWriterBuilder.java

```diff
@@ -59,6 +62,8 @@
   protected int queueCapacity = AbstractAsyncDataWriter.DEFAULT_BUFFER_CAPACITY;
   @Getter
   protected int maxAttempts = AsyncHttpWriter.DEFAULT_MAX_ATTEMPTS;
+  @Getter
+  protected SharedResourcesBroker<GobblinScopeTypes> broker = null;
```

Comment:
This should probably be `final`, as well as most of the instance variables in this method.